_docs/using-beaker/index.md

```diff
@@ -1,28 +1,224 @@
 ---
-title: The peer-to-peer Web
+title: Welcome to Beaker
 section: usingBeaker
 sectionTitle: Using Beaker
 order: 1
+class: dark
 ---
 
-<img data-src="/img/docs/tour-intro.png" class="centered">
+Beaker is a new kind of browser that gives you the power to create websites,
+share files, and control your data. It's an evolving reimagination of the Web,
+and we're excited to have you on the journey with us. Welcome!
 
-Peer-to-peer websites are just like any other website. But instead of being served from a datacenter, files are hosted by peers on your network. This makes it easy for anybody to host a site.
+<img data-src="/img/logo/256x256.png" style="display: block; margin: 3em auto">
+
+<section markdown="1">
+
+## A peer-to-peer Web
+
+At the heart of Beaker is a **peer-to-peer network** for sharing all kinds of
+content.
+
+It lets you skip the middle man and share data directly with your
+friends and colleagues. P2P is fantastic for sharing information privately, and
+it works great for publishing to a broad audience too!
 
 <figure>
 <img data-src="/img/docs/tour-site-peer-count.jpg">
-<figcaption>Browsing to a peer-to-peer site shows the number of peers</figcaption>
+<figcaption>Peer-to-peer networking skips the middle man and shares data directly.</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Using Beaker
+
+### Share personal files
+
+Create a site, add the files, and send the URL using a chat program. It's
```

Comment:
Ok, but I do think it's a bit odd to tell the user to use a chat program. Maybe we can just say "send the URL to your recipient".

At the very least, I think we need to fix one thing. "Send the URL to your recipient" not just "Send the URL"